onmt/bin/train.py

```diff
@@ -53,6 +52,33 @@ def prepare_fields_transforms(opt):
     return fields, transforms_cls
 
 
+def prepare_update_fields_transforms(opt, old_fields):
+    """Prepare or dump fields & transforms before training."""
+    transforms_cls = get_transforms_cls(opt._all_transform)
+    specials = get_specials(opt, transforms_cls)
+    fields = build_update_dynamic_fields(
+        opt, old_fields, src_specials=specials['src'], tgt_specials=specials['tgt'])
+
+    # maybe prepare pretrained embeddings, if any
+    prepare_pretrained_embeddings(opt, fields)
```

Comment:
Not compatible with the update, or is it?

Reply:
Not sure. I'll give it thought